java/src/main/java/gov/doc/ntia/sigmf/ext/global/diagnostics/Diagnostics.java

```diff
@@ -0,0 +1,64 @@
+package gov.doc.ntia.sigmf.ext.global.diagnostics;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Date;
+
+public class Diagnostics {
+
+  protected Computer computer;
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSX")
+  private Date datetime;
+  private Preselector preselector;
+  private SPU spu;
+  @JsonProperty(value = "action_runtime")
+  private Double actionRuntime;
+
+  public Date getDatetime() {
+    return datetime;
+  }
+
+  public void setDatetime(Date datetime) {
+    this.datetime = datetime;
+  }
+
+  public Preselector getPreselector() {
+    return preselector;
+  }
+
+  public void setPreselector(Preselector preselector) {
+    this.preselector = preselector;
+  }
+
+  public SPU getSpu() {
+    return spu;
+  }
+
+  public void setSpu(SPU spu) {
+    this.spu = spu;
+  }
+
+  public Computer getCpu() {
+    return computer;
+  }
+
+  public void setCpu(Computer computer) {
```

Comment:
remove `getCpu()` and `setCpu()` (duplicates of `getComputer()` and `setComputer()`

Reply:
Fixed in 1594a3b99d6d798128428125a962e3d6520232e4